static/elements/chromedash-myfeatures.js

```diff
@@ -0,0 +1,84 @@
+import {LitElement, html} from 'lit-element';
+import {nothing} from 'lit-html';
+import '@polymer/iron-icon';
+
+class ChromedashMyFeatures extends LitElement {
+  static get properties() {
+    return {
+      signedIn: {type: Boolean},
+      canEdit: {type: Boolean},
+      canApprove: {type: Boolean},
+      loginUrl: {type: String},
+      starredFeatures: {type: Object}, // will contain a set of starred features
+    };
+  }
+
+  constructor() {
+    super();
+    this.starredFeatures = new Set();
+    this.canEdit = false;
+    this.canApprove = false;
+  }
+
+  // Handles the Star-Toggle event fired by any one of the child components
+  handleStarToggle(e) {
+    const newStarredFeatures = new Set(this.starredFeatures);
+    window.csClient.setStar(e.detail.feature, e.detail.doStar)
+      .then(() => {
+        if (e.detail.doStar) {
+          newStarredFeatures.add(e.detail.feature);
+        } else {
+          newStarredFeatures.delete(e.detail.feature);
+        }
+        this.starredFeatures = newStarredFeatures;
+      })
+      .catch(() => {
+        alert('Unable to star the Feature. Please Try Again.');
+      });
+  }
+
+
+  renderBox(title, query, columns) {
+    return html`
+      <chromedash-accordion
+        title="${title}"
+        opened>
+
+        <chromedash-feature-table
+          query="${query}"
+          ?signedin=${this.signedIn}
+          ?canedit=${this.canEdit}
+          ?canApprove=${this.canApprove}
+          .starredFeatures=${this.starredFeatures}
+          @star-toggle-event=${this.handleStarToggle}
```

Comment:
Is this how you listen to the `'star-toggle-event` event?

Reply:
Yes, the @ syntax adds an event listener.  
https://lit-element.polymer-project.org/guide/events#add-declarative-event-listeners